### PR TITLE
HACDOCS-545: Update Add components Learn more based on current UI

### PR DESCRIPTION
**Pull request by jseseCCS**
[HACDOCS-545](https://issues.redhat.com/browse/HACDOCS-545)

I checked the current content for the **Add components Learn more** and it needs some minor changes to match the UI.

### CODE REVIEW

**Review by Hyperkid123 — APPROVED** [review]

**Comment by jseseCCS** [comment]
@Hyperkid123 Martin? Could you add Allison Wolfe as a reviewer for this PR (and the repo, too, for future PRs)?

**Comment by Hyperkid123** [comment]
> @Hyperkid123 Martin? Could you add Allison Wolfe as a reviewer for this PR (and the repo, too, for future PRs)?

Sure thing. Can you send me the GH username?

**Comment by jseseCCS** [comment]
yes. it's chloeredhat

thank you, @Hyperkid123!

**Comment by jseseCCS** [comment]
@Hyperkid123 thanks! yes as soon as she's online i'll ask her what it is. i tried to find it myself in other repos but can't. stand by!

**Comment by jseseCCS** [comment]
@Hyperkid123 Martin, it's wolfeallison. Thank you!

**Comment by jseseCCS** [comment]
making sure you saw [this](https://github.com/RedHatInsights/quickstarts/pull/118#issuecomment-1680537480), kind sir! @Hyperkid123

File: docs/help-topics/stonesoup-import/stonesoup-import.yml [hunk 1]
```diff
@@ -7,14 +7,13 @@
...
 
     **Using code from your GitHub repository**
-    1. To use your own code, enter the link to your repository in the **Git repo URL** field. We verify your URL right away. You don’t even have to press **Enter**.
-    2. Click **Git options** to expand it and choose whether to make either of the following specifications:
-        1. Add a **Git reference** to point to code in a specific branch, tag, or commit that you want to use to build your application.
-        2. Indicate a **Context dir** to specify the subdirectory for the source code you want to use.
-    3. Click **Next**, then we scan and analyze your Git directory to determine how many components to add to your application. The types of files in your repository, and also your Git directory file hierarchy, determine the number and type of components to add. For example, we look for devfiles and Dockerfiles, and language-specific build configuration files like pom.xml for Java or package.json for Node.js.
+    1. To use your own code, enter the link to your repository in the **Git repository URL** field. We verify your URL right away. You don’t even have to press **Enter**.
```
Comment (wolfeallison) [inline docs/help-topics/stonesoup-import/stonesoup-import.yml:13]: I could be overthinking this, but do we think it is confusing to tell the user to enter their link and then tell them they don't have to press enter? 

"To use your own code, enter the link to your repository in the **Git repository URL** field. We verify your URL as soon as it is entered, so you don't need to press anything." --> not sure if I like this either but brainstorming. WDYT @jseseCCS

File: docs/help-topics/stonesoup-import/stonesoup-import.yml [hunk 2]
```diff
@@ -7,14 +7,13 @@
...
+    2. Add an optional **Git reference** to point to code in a specific branch, tag, or commit that you want to use to build your application.
+    3. Indicate a **Context directory** to specify the subdirectory for the source code you want to use.
+    4. Click **Next**, then we scan and analyze your Git directory to determine how many components to add to your application. The types of files in your repository, and also your Git directory file hierarchy, determine the number and type of components to add. For example, we look for devfiles and Dockerfiles, and language-specific build configuration files like `pom.xml` for Java or `package.json` for Node.js.
```
Comment (wolfeallison) [inline docs/help-topics/stonesoup-import/stonesoup-import.yml:16]: For the second sentence, what do you think of making it a little more concise -- "The types of files in your repository and your Git directory file hierarchy determine the number and type of components to add." @jseseCCS

File: docs/help-topics/stonesoup-import/stonesoup-import.yml [hunk 3]
```diff
@@ -30,32 +29,51 @@
...
+
+    - **Component name**: Is it correct? Click the GitHub link under this field if you want to check your repository URL.
+    - **Runtime**: Did we detect the right one? If so, you should notice that we suggest appropriate deployment options. If you want to change your runtime, expand the menu next to your runtime name and make a selection.
 
-    Before you create and deploy your application, look over the values that display here. You can rename any component from this view, too. In particular, pay attention to the following fields:
+    Under **Build & deploy configuration**, review these settings:
 
     - **Target port**: Is it correct? If not, click in the port field to modify it.
-    - **CPUs**, **memory**, and **instances**: Choose how many of each of these you want for your application, depending on your subscription.
+    - **Dockerfile**: If you specified **Dockerfile** as your runtime, make sure we detected the right one. Click in the field if you need to modify it. Don't worry! We hide this field if you chose a different runtime.
```
Comment (wolfeallison) [inline docs/help-topics/stonesoup-import/stonesoup-import.yml:55]: I don't think we need the "Don't worry!" here.  Maybe just "If you chose a different runtime, we hide this field." @jseseCCS

**Comment by jseseCCS** [comment]
@wolfeallison thank you so much! i made all of your changes!!

**Comment by jseseCCS** [comment]
@Hyperkid123 can you remind me who publishes these changes to RHTAP?

**Comment by jseseCCS** [comment]
thanks, @Hyperkid123! if i do nothing, these changes will get published soon anyway, no?

**Comment by Hyperkid123** [comment]
We will be publish production update later today.

**Comment by jseseCCS** [comment]
> 

sounds great, @Hyperkid123. i don't see the update yet but i'll keep checking. please lmk if it'll be delayed past today. thanks!

**Comment by Hyperkid123** [comment]
Interesting it should be deployed.

**Comment by jseseCCS** [comment]
> Interesting it should be deployed. The image was bumped about 23h ago.

maybe i'm looking in the wrong place?? https://console.dev.redhat.com/preview/application-pipeline

**Pull request merged**